CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
 project(CMaizeSuperbuild LANGUAGES CXX)
 
+# Required to enable unit testing
+include(CTest)
+enable_testing()
+
 set(CMAKE_CXX_STANDARD 11)
```

Comment:
You aren't using an C++11 features, so don't force it. Anyways the C++ standard should be set on a target not globally. Also you should never call `set()` for variables defined by CMake (such as you do on lines 9 to 11); doing so will override the user's options. Usually you want to use the `option` command (which will only set them if the user hasn't).